tests/installation/test_pip_installer.py

```diff
@@ -101,7 +101,9 @@ def test_requirement_git_subdirectory(
     installer: PipInstaller, package_git_with_subdirectory: Package
 ):
     result = installer.requirement(package_git_with_subdirectory)
-    expected = "git+https://github.com/python-poetry/demo.git@master#egg=demo&subdirectory=subdir"
+    expected = (
+        "git+https://github.com/poetry/demo.git@master#egg=demo&subdirectory=subdir"
```

Comment:
I wouldn't call this better since while we're not actually hitting that URL, it's close enough to a real one (while not being correct) that someone might be misled.

To avoid the E501, I'd suggest just using a multiline string as shown in https://stackoverflow.com/a/5437652.